macosx/en.lproj/InfoPlist.strings

```diff
@@ -1,3 +1,4 @@
 /* Localized versions of Info.plist keys */
 
 NSHumanReadableCopyright = "Copyright © 2005-2024 The Transmission Project";
+MDItemKeywords = "tor,torrent,torrents,magnet,magnets,bt,bit,btih";
```

Comment:
1. `tor` seems unrelated, see https://www.torproject.org/
2. Do we need plural forms for `torrent` and `magnets`?
3. What are `bt`, `bit`, `btih` for? Are they for `magnet:?` query params? Isn't it already covered by `magnet` keyword itself?

Reply:
It doesn't matter if tor is unrelated: users of spotlight only want to write 2 or 3 characters to find an app, so here "tor" is short for torrent.
**Spotlight keywords are with exact matches only.** So you need to declare both singular and plural if you want discoverability for both.
bt and bit are short for bittorrent, the protocol
btih is the magnet protocol

Reply:
1. Will Spotlight prioritize "Tor Browser" over "Transmission" in this case?
2. Okay, but I just relied on [usages](https://github.com/search?q=%22MDItemKeywords%22+AND+%28+path%3A*.plist+OR+path%3A*.strings+%29+AND+NOT+is%3Afork&type=code) in other projects.

So, aside from `magnet` and `torrent`, this seems like aggressive SEO strategies :D

Reply:
Spotlight offers a list of results, and it prioritizes app names (so Tor Browser will show before Transmission).
<img width="712" alt="Capture d’écran 2024-02-13 à 21 23 58" src="https://github.com/transmission/transmission/assets/839992/d997b165-10cc-4937-b2de-edfe5ac67c26">

I'll remove btih which is too obscure.